### PR TITLE
doc: update changelogs for release

### PR DESCRIPTION
**Pull request by b-gent**
Change 'Master branch' heading to 'nRF Connect SDK v1.4.0'.

Affected changelogs:

- SoftDevice Controller
- MPSL
- NFC

Signed-off-by: Bartosz Gentkowski <bartosz.gentkowski@nordicsemi.no>

### CODE REVIEW

**Review by rugeGerritsen — APPROVED** [review]

**Review by joerchan — APPROVED** [review]

**Comment by b-gent** [comment]
MAnifest updated in https://github.com/nrfconnect/sdk-nrf/pull/3112

**Comment by b-gent** [comment]
Adding DNM now as this should go in right before the tag.

**Review by shantha-14 — APPROVED** [review]

**Comment by joerchan** [comment]
@b-gent This is a pretty significant change in the contents of the PR. I think you should dismiss approvals when things like this happens.

**Review by rugeGerritsen — APPROVED** [review]

**Comment by joerchan** [comment]
@rugeGerritsen Needs to be updated source repos

**Review by joerchan — CHANGES_REQUESTED** [review]
~~Conflicts with bug-fix of SoftDevice controller now.~~

~~Known issues needs to be updated with the contents of the changelog here:
https://github.com/nrfconnect/sdk-nrfxlib/pull/303~~

**Review by joerchan — APPROVED** [review]

**Comment by b-gent** [comment]
> @b-gent This is a pretty significant change in the contents of the PR. I think you should dismiss approvals when things like this happens.

Agreed.

File: doc/links.txt [hunk 1]
```diff
@@ -22,7 +22,8 @@
 
 .. _`Socket API`: https://pubs.opengroup.org/onlinepubs/9699919799/functions/socket.html
 
-.. _`ZBOSS API documentation`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/zboss/index.html
+.. _`ZBOSS API documentation`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/zboss_api_doc/index.html
+.. _`ZBOSS stack v3.3.0.4 release notes`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/zboss/3.3.0.4/zboss_release_notes.html
```
Comment (tomchy) [inline doc/links.txt:26]: If this PR gets merged: https://github.com/nrfconnect/sdk-nrf/pull/3205
Please update the link name /or/ move the doc upload and change both: the link name and the URL.

File: zboss/CHANGELOG.rst [hunk 2]
```diff
@@ -21,6 +21,7 @@ Changes
 =======
 
 * Updated ZBOSS stack to version ``10_06_2020``.
+  For detailed release notes for this stack version, see `ZBOSS stack v3.3.0.4 release notes`_.
```
[the remark this answers is not in the document]
Comment (greg-fer) [reply zboss/CHANGELOG.rst:24]: We have two versions of doc sets for ZBOSS now, 3.3.0.3 and 3.3.0.5, so v3.3 is not good enough.

File: doc/links.txt (hunk 1 above)
Comment (greg-fer) [reply doc/links.txt:26]: @b-gent , can you help with this?

Comment (greg-fer) [reply doc/links.txt:26]: @b-gent , can you help with updating the directory name nad the landing page?

Comment (b-gent) [reply doc/links.txt:26]: Do we really need to mention the version in this link? The idea of having a separate HTML page was to maintain all version information there, without worrying about it in NCS links.

File: zboss/CHANGELOG.rst (hunk 2 above)
[the remark this answers is not in the document]
Comment (b-gent) [reply zboss/CHANGELOG.rst]: Do we really need to mention the version in this link? The idea of having a separate HTML page was to maintain all version information there, without worrying about it in NCS links.

[the remark this answers is not in the document]
Comment (greg-fer) [reply zboss/CHANGELOG.rst]: As discussed during call, we can remove version from here as long as we add release note links on ZBOSS doc landing page.

**Review by tomchy — APPROVED** [review]

**Review by greg-fer — APPROVED** [review]

**Review by grochu — APPROVED** [review]

**Review by anangl — APPROVED** [review]

**Pull request merged**